Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 version = "0.2.0"
 
 [dependencies]
-crossterm = { version = "0.19.0", default-features = false }
+crossterm = "0.20.0"
```

Comment:
I also changed the version declaration for more simplicity. So far, the [default feature of crossterm](https://github.com/crossterm-rs/crossterm/blob/a618a9f3b35dfbaf092c5b7dbbbe48b646208953/Cargo.toml#L29) seems empty.